core/sql/arkcmp/CmpStatement.cpp

```diff
@@ -1254,17 +1255,34 @@ CmpStatement::process(const CmpMessageDatabaseUser &statement)
 {
   NABoolean doDebug = FALSE;
 
-  NAString message = statement.data();
+  // The message contains the following:
```

Comment:
Should we check the length of the message to make sure it covers the various fields checked here?